src/Magento/FunctionalTestingFramework/Test/Util/TestObjectExtractor.php

```diff
@@ -155,4 +170,34 @@ public function extractTestData($testData)
             throw new XmlException($exception->getMessage() . ' in Test ' . $filename);
         }
     }
+
+    /**
+     * Append names of test files, including merge files, in description annotation
+     *
+     * @param string $description
+     * @param array  $fileNames
+     *
+     * @return string
+     */
+    private function appendFileNamesInDescriptionAnnotation($description, $fileNames)
+    {
+        $title = '';
+        foreach ($fileNames as $fileName) {
+            if (!empty($fileName && realpath($fileName) !== false)) {
+                $fileName = realpath($fileName);
+                $relativeFileName = ltrim(
+                    str_replace(MAGENTO_BP, '', $fileName),
+                    DIRECTORY_SEPARATOR
+                );
+                if (!empty($relativeFileName)) {
+                    if (empty($title)) {
+                        $title .= '<br><br>Test Files:<br>';
```

Comment:
Asked around for opinions, we should bold the `<b>Test Files:</b>` for better readability.

Reply:
It's obvious that bold is better. I will just go ahead change it